src/main/java/net/imagej/ops/filter/derivative/PartialDerivativesRAI.java

```diff
@@ -0,0 +1,51 @@
+package net.imagej.ops.filter.derivative;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.imagej.ops.Ops;
+import net.imagej.ops.Ops.Filter.AllPartialDerivatives;
+import net.imagej.ops.special.chain.RAIs;
+import net.imagej.ops.special.computer.UnaryComputerOp;
+import net.imagej.ops.special.function.AbstractUnaryFunctionOp;
+import net.imagej.ops.special.function.UnaryFunctionOp;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.type.numeric.RealType;
+import net.imglib2.view.Views;
+import net.imglib2.view.composite.CompositeIntervalView;
+import net.imglib2.view.composite.GenericComposite;
+import net.imglib2.view.composite.RealComposite;
+
+import org.scijava.plugin.Plugin;
+
+@Plugin(type = Ops.Filter.AllPartialDerivatives.class, name = Ops.Filter.AllPartialDerivatives.NAME)
+public class PartialDerivativesRAI<T extends RealType<T>, C extends GenericComposite<T>>
+		extends AbstractUnaryFunctionOp<RandomAccessibleInterval<T>, CompositeIntervalView<T, RealComposite<T>>>
```

Comment:
The output could be "generalized" to a `RandomAccessibleInterval<? extends Composite<T>>`.